deployment/terraform/create.go

```diff
@@ -1118,6 +1118,7 @@ func (t *Terraform) init() error {
 	assets.RestoreAssets(t.config.TerraformStateDir, "keycloak.service")
 	assets.RestoreAssets(t.config.TerraformStateDir, "saml-idp.crt")
 	assets.RestoreAssets(t.config.TerraformStateDir, "provisioners")
+	assets.RestoreAssets(t.config.TerraformStateDir, "saml-idp.crt")
```

Comment:
Wasn't this already fixed in https://github.com/mattermost/mattermost-load-test-ng/commit/7f6be66fa3abb1ef4fe9636d24ed3b7145d8a4c8?

Reply:
I may had that in my local from testing, removed 👍